spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -863,5 +863,107 @@ def expect_sync_cloud_tenants_with_tenants_is_queued
       expect(sync_cloud_tenant.method_name).to eq("sync_cloud_tenants_with_tenants")
       expect(sync_cloud_tenant.state).to eq(MiqQueue::STATE_READY)
     end
+
+    def assert_targeted_vm(vm_name, attributes)
```

Comment:
it will require some tweaking, yo ucan also do more API queries to fetch e.g. the network_ports, given you will be able to filter that API using e.g. a Vm uuid